src/tracks/playabletrack/wavetrack/ui/ClipOverflowButtonHandle.cpp

```diff
@@ -0,0 +1,103 @@
+#include "ClipOverflowButtonHandle.h"
```

Comment:
I could fix the first commit so it compiles -- but then the second fails to link!  You neglected to add the new files to `CMakeLists.txt`.  Also the new files should have banners.

Reply:
After I fix CMakeLists.txt, still, to build I must add:

```
#include <wx/window.h>
```